test/Microsoft.Framework.Runtime.FunctionalTests/ServiceBreadcrumbsFacts.cs

```diff
@@ -31,7 +31,6 @@ public void BreadcrumbsAreCreatedSuccessfully()
             breadcrumbs.AddBreadcrumb("Test", new NuGet.SemanticVersion("1.0.0"));
             breadcrumbs.WriteAllBreadcrumbs();
 
-            Assert.True(File.Exists(Path.Combine(TempFolderPath, "Test")));
             Assert.True(File.Exists(Path.Combine(TempFolderPath, "Test.1.0.0")));
```

Comment:
Is it possible to add a test that verifies that runtime breadcrumb is being written?